Cairo Desktop/Cairo Desktop/Cairo.xaml

```diff
@@ -83,6 +84,8 @@
                       Direction="90"
                       Opacity="0.6"
                       BlurRadius="0" />
+    <Thickness x:Key="MenuBarBorderThickness" Right="1" />
```

Comment:
Pretty sure this should be bottom, not right. Can you adjust?

Reply:
Hmmm, ill take a look